client/src/store/modules/comments.js

```diff
@@ -0,0 +1,70 @@
+import Vue from "vue"
+import { firebaseDb } from "../../config/config.js"
+
+let messagesRef
+
+const state = {
+  userDetails: {
+    name: "Happy emOcean User",
+    email: "johndoe@gmail.com",
+    uid: '123'
+    // uid: "456"
+  },
+  messages: {}
+}
+const mutations = {
+  addMessage (state, payload) {
+    Vue.set(state.messages, payload.messageId, payload.messageDetails)
+  },
+  removeMessage (state, payload) {
+    Vue.delete(state.messages, payload.messageId)
+  },
+  clearMessages (state) {
+    state.messages = {}
+  }
+}
+const actions = {
+  firebaseGetMessages ({ commit, state }, postId) {
```

Comment:
Why function with getMessages contains logic with addMessage to store?
Why one action corresponding for addMessage and removeMessage?